src/server/render.js

```diff
@@ -29,15 +28,10 @@ export default stats => async (req, res) => {
 
     const root = App(store, StaticRouter, { location: req.url, context })
 
-    const page = (
-      <Html
-        stats={stats}
-        state={store.getState()}
-        content={renderToString(root)}
-      />
-    )
+    const page = <Html stats={stats} state={store.getState()} content={renderToString(root)} />
```

Comment:
I've configured prettier to "cut" at 100 columns. Do you think 80 would be better?